src/controllers/enter.information.controller.ts

```diff
@@ -2,12 +2,12 @@ import { Session } from "ch-node-session-handler";
 import { NextFunction, Request, Response } from "express";
 import ObjectionCompanyProfile from "model/objection.company.profile";
 import { SESSION_COMPANY_PROFILE, SESSION_OBJECTION_ID } from "../constants";
+import * as PageUrls from "../model/page.urls";
```

Comment:
would it be better to just import the page urls you need rather than importing them all?

Reply:
I can switch them, it read better to me this way.